pyproject.toml

```diff
@@ -23,11 +23,12 @@ maintainers = [
 
 dependencies = [
     "arviz_plots @ git+https://github.com/arviz-devs/arviz-plots",
-    "formulae>=0.5.3",
+    "formulae @ git+https://github.com/bambinos/formulae",
     "graphviz",
     "matplotlib>=3.9.0",
     "pandas>=1.0.0",
     "pymc>=5.26.1",
+    "sparse>=0.17.0,<0.18",
```

Comment:
xarray does not work with sparse matrices from scipy, thus we need this extra dependency.